Added CSS to make it look pretty

ngkiddle opened the pull request:
also had to adjust some HTML to get the CSS implimented

lemme know if this looks good enough to merge with master!

ngkiddle committed:
Added CSS to make it look pretty

also had to adjust some HTML to get the CSS implimented

ngkiddle commented:
Also this uses his working code that was provided for sprint 2

michaelabuchanan commented:
Tried it out on my computer (Firefox and Chrome) and it looks a lot better than before! The ship placing worked for me now too which is always good. A few small things if you have time for it: I was wondering if we could vertically center the grid on the screen. Also if maybe we could change the color of the ship we are placing so we know which one we chose? Not 100% necessary since you can tell from the number of squares that show up but might be nice. Also the background image is a bit too wide for the screen. If you don't have time to implement these, feel free to let me know and I might have time to do some of it since I only have the cancel buttons. For the future, it might be good to implement a layout that would work better on more narrow screens so the grid and the ships don't overlap. However, since we decided that a 16:9 laptop screen is our target platform for this I don't think we have to worry about it for this sprint. Everything looks good to me otherwise.

ngkiddle committed:
made pull request changes

Ships are now color coded when selecting and placing, no more horizontal scroll aka fits to window. The grids look more centered vertically not sure why tho... but they do!

ngkiddle committed:
Design Drawings
